### PR TITLE
Add an example of database connection

### PR DESCRIPTION
**Pull request by AA666666**

### CODE REVIEW

**Comment by AA666666** [comment]
The final change is done. Please review it when you have time. Thanks.

**Review by Jaredhw — DISMISSED** [review]

File: database connection using node example.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    console.log("Connected!");
+    con.query(sql, function (err, result) {
+      if (err) throw err;
+      console.log("Result: " + result);
+    });
+  });
+
+  //Creating a Database
+
+  var mysql = require('mysql');
```
Comment (JadenBalogh) [inline database connection using node example.js:32]: You only need to use `require()` once for each package you are importing.

File: database connection using node example.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+var con = mysql.createConnection({
+  host: "localhost",
+  user: "yourusername",
+  password: "yourpassword"
+});
```
Comment (JadenBalogh) [inline database connection using node example.js:38]: You should only have to create 1 connection to the database in this file.

**Review by JadenBalogh — APPROVED** [review]
We'll consider this file as practice for creating database connections in Node.js, and use the demodb.js file as reference for creating these connections in the future.

**Review by Jaredhw — APPROVED** [review]

**Pull request merged**